spack-repo/packages/ports-of-call/package.py

```diff
@@ -9,18 +9,41 @@
 class PortsOfCall(CMakePackage):
     """Ports of Call: Performance Portability Utilities"""
 
-    homepage    = "https://github.com/lanl/ports-of-call"
-    url         = "https://github.com/lanl/ports-of-call/archive/refs/tags/v1.1.0.tar.gz"
-    git         = "https://github.com/lanl/ports-of-call.git"
+    homepage = "https://github.com/lanl/ports-of-call"
+    url = "https://github.com/lanl/ports-of-call/archive/refs/tags/v1.1.0.tar.gz"
+    git = "https://github.com/lanl/ports-of-call.git"
 
-    maintainers = ['rbberger']
+    maintainers = ["rbberger"]
 
     version("main", branch="main")
-    version("1.4.2", sha256="02e0ae66cda5bec7c2121d16728b9fbaa653b3863d11737e43b303145ddb2cb2")
+    version("1.5.1", sha256="b1f0232cd6d2aac65385d77cc061ec5035283ea50d0f167e7003eae034effb78")
     version("1.4.1", sha256="82d2c75fcca8bd613273fd4126749df68ccc22fbe4134ba673b4275f9972b78d")
     version("1.4.0", sha256="e08ae556b7c30d14d77147d248d118cf5343a2e8c0847943385c602394bda0fa")
```

Comment:
It doesn't matter much here, since these are only used for CI and local testing. But the question is relevant for the Spack upstream version of this spackage. In general, if you want to tell users not to use a particular version anymore in Spack, you deprecate it for one Spack release and remove it in the next.

Reply:
Ah I see. My question was motivated by what appeared to be different treatment for `ports-of-call` and `spiner`. In the latter's case, all but the most recent release are being deprecated.